.dotnet/src/Generated/Files.cs

```diff
@@ -564,6 +564,11 @@ public virtual ClientResult DownloadFile(string fileId, RequestOptions options)
             try
             {
                 using PipelineMessage message = CreateDownloadFileRequest(fileId, options);
+
+                // TODO: If we want the end-user to be able to Stream the response, we'll
+                // need to update this somehow to apply that pattern.  Let's sync on the
+                // best approach for implementing this in this client.
```

Comment:
Flagging the protocol method corresponding to the changed convenience method.

Reply:
Resolving comment since the experimental "Stream output" approach has been removed from the PR.